lib/helpers.nix

```diff
@@ -140,9 +140,9 @@ with lib; rec {
       ''
     );
 
-  defaultNullOpts = let
-    maybeRaw = t: lib.types.either t rawType;
-  in rec {
+  maybeRaw = t: lib.types.either t rawType;
+
+  defaultNullOpts = rec {
```

Comment:
maybe put that in the newly created `nixvimTypes`

Reply:
Like so ?